nuget/spec/dependabot/nuget/update_checker/version_finder_spec.rb

```diff
@@ -671,7 +677,7 @@ def create_nupkg(nuspec_name, nuspec_content)
     its([:version]) { is_expected.to eq(version_class.new("2.0.0")) }
 
     context "when the user is ignoring the lowest version" do
-      let(:ignored_versions) { [">= 2.a, <= 2.0.0"] }
```

Comment:
This test was originally incorrect for the same reason as the one above so it has been simplified.